src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -410,8 +409,13 @@ protected virtual void Initialize()
                 }
 
                 // Load builtin extensions
-                var sendGridExtension = new Extensions.SendGrid.SendGridConfiguration();
-                LoadExtension(sendGridExtension);
+                {
+                    var botExtension = new Extensions.BotFramework.Config.BotFrameworkConfiguration();
```

Comment:
Have you made sure all the host level config knobs on BotFrameworkConfiguration still are settable via host.json after the replat?